proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/presenter/DigitalObjectManager.java

```diff
@@ -303,6 +310,8 @@ private void initContextMenu(Menu menu, ActionSource actionSource) {
         menu.addItem(new MenuItemSeparator());
         menu.addItem(Actions.asMenuItem(deleteAction, actionSource, true));
         menu.addItem(Actions.asMenuItem(registerUrnNbnAction, actionSource, true));
+        menu.addItem(new MenuItemSeparator());
+        menu.addItem(Actions.asMenuItem(expandTreeAction, actionSource, false));
```

Comment:
@pokorsky 
kontrakt volané metody je
```
asMenuItem(final Action action, final Object source, boolean willAsk)
```
intuitivně jsem očekával, že je to u položek, které se budou ptát na potvrzení, ale není to pravda. Tam kde je předáno true, tak se pouze do tooltipu přidají tři tečky. @pokorsky Přijde mi to matoucí, je to chyba?